be/etp-backend/src/main/java/com/psa/ExchangeGrpcImpl.java

```diff
@@ -158,17 +156,14 @@ public void getOrders(
                 Integer quantity = resultSet.getInt("quantity");
                 Integer quantityFilled = resultSet.getInt("quantity_filled");
                 Integer quantityCancelled = resultSet.getInt("quantity_cancelled");
-                response.addOrders(Order.newBuilder()
-                        .setId(id)
-                        .setStatus(status)
-                        .setBroker(broker)
-                        .setSymbol(symbol)
-                        .setSide(side)
-                        .setType(type)
-                        .setPrice(price)
-                        .setQuantity(quantity)
-                        .setQuantityFilled(quantityFilled)
-                        .setQuantityCancelled(quantityCancelled)
+//                java.sql.Timestamp sqlTimestamp = resultSet.getTimestamp("timestamp");
+//
+//                Timestamp protoTimestamp = Timestamp.newBuilder()
+//                        .setSeconds(sqlTimestamp.getTime() / 1000) // Convert to seconds
+//                        .setNanos(sqlTimestamp.getNanos())         // Set nanoseconds
+//                        .build();
```

Comment:
Feel free to work on this in your next PR, so you can merge the rest of the changes.